src.mk

```diff
@@ -371,6 +372,7 @@ MAIN_SOURCES =                                                          \
   tools/reduce_levels_test.cc                                           \
   tools/sst_dump_test.cc                                                \
   tools/trace_analyzer_test.cc						\
+  trace_replay/block_cache_tracer_test.cc						                    \
```

Comment:
You may also want to adjust the alignment of `\`. It's not strictly required, but other lines are trying to align.

Reply:
My bad. I changed my editor to show indentation now. I updated it.